torchvision/datasets/cityscapes.py

```diff
@@ -127,8 +128,19 @@ def __init__(self, root, split='train', mode='fine', target_type='instance',
                              ' or "color"')
 
         if not os.path.isdir(self.images_dir) or not os.path.isdir(self.targets_dir):
-            raise RuntimeError('Dataset not found or incomplete. Please make sure all required folders for the'
-                               ' specified "split" and "mode" are inside the "root" directory')
+            image_dir_zip = os.path.join(self.root, 'leftImg8bit') + '_trainvaltest.zip'
```

Comment:
@RJT1990 this is missing case were we're using the `coarse` set with `train_extra` split. In that case the additional images zip is called `leftImg8bit_trainextra.zip`. Maybe you could add that case as well in a follow up pr?